debugger/src/dconfig.c

```diff
@@ -308,11 +318,20 @@ static gpointer saving_thread_func(gpointer data)
 			debug_config_changed = FALSE;
 		}
 
+#if GTK_CHECK_VERSION(3, 0, 0)
+		interval = g_get_monotonic_time() + SAVING_INTERVAL;
+#else
```

Comment:
No gtk dependant code. It depends on the glib version. What is the reason for this change? If it is an improvement then the code could be used without any check as the function ```g_get_monotonic_time()``` exists since glib 2.28 and geany requires 2.32.

Reply:
This change has been made to use the new `g_cond_wait_until()` function which replaces the deprecated `g_cond_timed_wait()` one. The old code has been replaced with the new one.